common/src/main/java/com/microsoft/identity/common/internal/commands/EmptyCallback.java

```diff
@@ -0,0 +1,40 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.internal.commands;
+
+import com.microsoft.identity.common.internal.result.EmptyResult;
+
+/*
+    Null Object Pattern for Commands who's result should be ignored.
+ */
+public class EmptyCallback implements CommandCallback<EmptyResult, Exception>{
```

Comment:
Why not Void?

Reply:
Void for?

Reply:
There is no default constructor available in the BaseCommand, and in order to extend it we need use one of its other constructors, which requires a @NonNull arg.

Reply:
I am not sure what we get with this type of empty callback..I think you want an actual implementation here where all that the implementation would do is that it would log the result for now

Reply:
It's not that we get anything, it's more of a limitation of how we designed BaseController. Logging makes sense.